x-pack/packages/security-solution/features/src/security/product_feature_config.ts

```diff
@@ -79,10 +79,17 @@ export const securityDefaultProductFeaturesConfig: DefaultSecurityProductFeature
 
   [ProductFeatureSecurityKey.endpointArtifactManagement]: {
     subFeatureIds: [
+      SecuritySubFeatureId.hostIsolationExceptions,
       SecuritySubFeatureId.trustedApplications,
       SecuritySubFeatureId.blocklist,
       SecuritySubFeatureId.eventFilters,
     ],
+  },
+
+  // Endpoint Complete Tier:
+  // Allows access to create/update HIEs
+  [ProductFeatureSecurityKey.endpointHostIsolationExceptions]: {
+    subFeatureIds: [SecuritySubFeatureId.hostIsolationExceptions],
```

Comment:
NIT: It's not strictly necessary to add the `subFeatureIds: [SecuritySubFeatureId.hostIsolationExceptions]` here since it's already included in _essentials_ tier by the `ProductFeatureSecurityKey.endpointArtifactManagement` config. 
Nevertheless, it does not have any bad implications either, _subFeatureIds_ are deduplicated internally. As you prefer.